src/digitalmarketplace/components/cookie-settings/cookie-settings.js

```diff
@@ -0,0 +1,129 @@
+// Javascript code to support the Cookie Settings page
+import { getCookie, setConsentCookie } from '../../helpers/cookie/cookie-functions'
+import InitialiseAnalytics from '../analytics/init'
+
+export function CookieSettings ($module) {
+  this.$module = $module
+}
+
+CookieSettings.prototype.init = function () {
+  this.$module.submitSettingsForm = this.submitSettingsForm.bind(this)
+
+  this.$module.addEventListener('submit', this.$module.submitSettingsForm)
+
+  // Ensure there aren't two forms for setting cookie preferences on the same page
+  this.hideCookieBanner()
+
+  this.setInitialFormValues()
+}
+
+CookieSettings.prototype.setInitialFormValues = function () {
+  var currentConsentCookie = getCookie('dm_cookies_policy')
+
+  if (!currentConsentCookie) {
+    // Don't populate the form
+    return
+  }
+
+  this.hideWarningMessage()
```

Comment:
These lines lead me to believe that if we visit this page without cookie settings, we should see a warning message; however, that doesn't happen for me 🤔

Reply:
> if we visit this page

Which page? I am still updating https://github.com/alphagov/digitalmarketplace-user-frontend/pull/214 so if you have that checked out on your User FE it won't line up just yet

Reply:
Ah okay. I was referring to the cookie settings page.